test-full-conversation.sh

```
@@ -0,0 +1,123 @@
+
+# Function to send a message and capture response
+send_message() {
+    local message="$1"
+    local turn="$2"
+    
+    echo ""
+    echo "💬 Turn $turn: $message"
+    echo "Response:"
+    
+    # Send the message and capture the streaming response
+    local response=$(curl -X POST "$BASE_URL" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)